src/query/functions/tests/it/scalars/testdata/array.txt

```diff
@@ -7,22 +7,23 @@ output domain  : []
 output         : []
 
 
-ast            : ['a', 1]
-raw expr       : array("a", 1_u8)
-checked expr   : array<T0=String><T0, T0>("a", to_string<UInt8>(1_u8))
-optimized expr : ["a", "1"]
-output type    : Array(String)
-output domain  : [{"1"..="a"}]
-output         : ["a", "1"]
+error: 
+  --> SQL:1:1
+  |
+1 | ['a', 1]
+  | ^^^^^^^^ unable to parse string to type `UInt8` because invalid digit found in string while evaluating function `to_uint8("a")`
```

Comment:
The arguments in array function should support cast from Integer to String.

Reply:
I had a discussion with @sundy-li yesterday, and we think it might be better to make string as the bottom type, to avoid potential ambigiouty.

Reply:
Yes, string is bottom type.